test/method/testorbitals.py

```diff
@@ -0,0 +1,53 @@
+# This file is part of cclib (http://cclib.github.io), a library for parsing
+# and interpreting the results of computational chemistry packages.
+#
+# Copyright (C) 2016 the cclib development team
+#
+# The library is free software, distributed under the terms of
+# the GNU Lesser General Public version 2.1 or later. You should have
+# received a copy of the license along with cclib. You can also access
+# the full license online at http://www.gnu.org/copyleft/lgpl.html.
+
+"""Test the various population analyses (MPA, LPA, CSPA) in cclib"""
+
+import sys
+import os
+import logging
+import unittest
+
+import numpy
+
+sys.path.append("..")
+from test_data import getdatafile
+from cclib.method import Orbitals
+from cclib.parser import Gaussian
+
+
+class RestrictedCalculationTest(unittest.TestCase):
+    """Check RHF calculation."""
+    def setUp(self):
+        self.data, self.logfile = getdatafile(Gaussian, "basicGaussian09", "dvb_sp.out")
+
+    def test_closed_shell(self):
+        self.assertTrue(Orbitals(self.data).closed_shell())
+
+
+class UnrestrictedCalculationTest(unittest.TestCase):
+    """Check RHF calculation."""
+    def setUp(self):
+        self.data, self.logfile = getdatafile(Gaussian, "basicGaussian09", "dvb_un_sp.log")
+
+    def test_closed_shell(self):
+        self.assertFalse(Orbitals(self.data).closed_shell())
+
+
+# TODO: add a case (regression) with an unrestricted calculation for a closed shell system.
```

Comment:
Added a comment. We would need to add this as a regression test (will do after this is in).